### PR TITLE
Update prettier to >= v2.0.5

### PR DESCRIPTION
**Pull request by BridgeAR**
# Update prettier

Closes https://github.com/MaibornWolff/codecharta/issues/982

## Definition of Done

A task/pull request will not be considered to be complete until all these items can be checked off.

- [x] **All** requirements mentioned in the issue are implemented
- [x] Does match the Code of Conduct and the Contribution file
- [x] Task has its own **GitHub issue** (something it is solving)
  - [ ] Issue number is **included in the commit messages** for traceability
- [ ] **Update the README.md** with any changes/additions made
- [ ] **Update the CHANGELOG.md** with any changes/additions made
- [ ] **Enough test coverage to ensure that uncovered changes do not break functionality**
- [x] **All tests pass**
- [x] **Descriptive pull request text**, answering:
  - What problem/issue are you fixing?
  - What does this PR implement and how?
- [ ] **Assign your PR to someone** for a code review
  - This person _will be contacted **first**_ if a bug is introduced into `master`
- [x] **Manual testing** did not fail

### CODE REVIEW

File: visualization/.prettierrc.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "printWidth": 140,
+    "printWidth": 120,
```
Comment (alschmut) [inline visualization/.prettierrc.json:2]: It seems `prettier` decided to overwrite this value. I personally would continue with the 140, but that's your decision :)

Comment (NearW) [reply visualization/.prettierrc.json:2]: I agree with @alschmut 
Maybe this will reduce the amount of changes in this PR :)

Comment (NearW) [reply visualization/.prettierrc.json]: I looked at the changes that 140->120 brought and I'm not a fan. Sometimes method headers and other lines get broken down, even thought they're much more readable with 140 line length. At least on my laptop I can easily display the full 140 characters.

I pushed some commits that changed the printWidth to 140 again and formatted the project accordingly. 
@BridgeAR Is that fine to you?

**Comment by BridgeAR** [comment]
@NearW it's your call. In what way are the handlebars files related though?

**Comment by NearW** [comment]
@BridgeAR When I formatted the project I got a warning because there was no formatter for handlebar-files defined. Getting rid of that warning was easy so I pushed the commit to that PR, since it does not add any complexity.

**Pull request merged**